python/cudf/cudf/tests/test_binops.py

```diff
@@ -377,7 +377,7 @@ def test_reflected_ops_scalar(func, dtype, obj_class):
     ps_result = func(random_series)
 
     # verify
-    np.testing.assert_allclose(ps_result, gs_result)
+    utils.assert_eq(ps_result, gs_result)
```

Comment:
It looks like in the case where the `obj_class` is `Index` that `gs_result` is a `cudf.Index` here while Pandas is a `pd.Series` and the equality check errors for that reason. I think the line 374 should be: `gs_result = Series(gs_result)`

Reply:
Thanks Keith! I'll take a closer look 🙂